app/scripts/controllers/provider-approval.js

```diff
@@ -1,177 +0,0 @@
-const ObservableStore = require('obs-store')
```

Comment:
@rekmarks We should probably do a migration to remove the parts of state associated with this controller

Reply:
Added migration `040` in 7538052